langchain/schema.py

```diff
@@ -394,6 +403,19 @@ def dict(self, **kwargs: Any) -> Dict:
         return output_parser_dict
 
 
+class DefaultOutputParser(BaseOutputParser[str]):
```

Comment:
Do we have a name that's better communicates that this is a passthrough parser? (e.g., PassthroughParser or NoOpParser?) i interpret default as its applied as the default, but does not reveal to me what it does.